tests/system/cases/windows_service/main.tf

```diff
@@ -7,7 +7,9 @@ provider "layer0" {
 resource "layer0_environment" "ws" {
   name = "ws"
   os   = "windows"
-  size = "m3.large"
+  scale = 1
```

Comment:
[nit] you can format terraform files with `terraform fmt`

Reply:
✅